openshift/templates/backup/backup-build.yaml

```diff
@@ -0,0 +1,72 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: "${NAME}-build-template"
+  creationTimestamp:
+objects:
+  - kind: ImageStream
+    apiVersion: v1
+    metadata:
+      name: "${NAME}"
```

Comment:
May want to append to the name, or by accident someone could choose a name that exists for a different configuration.